.github/CONTRIBUTING.md

```diff
@@ -1,16 +1,110 @@
-## OpenCollar needs your help!
+# Contributing to OpenCollar
```

Comment:
This document overlaps a lot with https://github.com/OpenCollarTeam/opencollarteam.github.io/pull/7.

We need to make sure we don't end up with two conflicting statements of project processes and policies.  I'd like to put the more long form stuff there and keep "CONTRIBUTING.md" to a more barebones summary of how we use Git.  Thoughts?

Reply:
I created these documents as part of the last meeting on 3/14/2021 and was not aware that anyone else was working on such a document. The topic was abruptly ended without making a final decision. Aria and many others (including myself) were in a quandary as to how to handle things. I have received occasional feedback from Aria and Roan during the writing of these guidelines to ensure that they are in line with OpenCollar's goal.

Regarding the seperate repo: My understanding is that all rules/policies for a given repository must be in the same repository. As an on-off contributor, I wouldn't want to check Nx different pages/repos just to figure out how and what to do - and under which policies/guidelines.